packages/maskbook/src/plugins/dHEDGE/define.tsx

```diff
@@ -57,6 +59,12 @@ export const DHedgePluginDefine: PluginConfig = {
 function Renderer(props: React.PropsWithChildren<{ url: string }>) {
     const DHEDGE_POOL_PATTERN = usePoolUrlPattern()
     const address = props.url.match(DHEDGE_POOL_PATTERN) || []
+
+    //#region check pool
+    const { value: pool, error, loading, retry } = useFetchPool(address[1])
```

Comment:
Found the same logic in the `<PoolView />`, why not do the validation in that component?